doc/cprover-manual/goto-harness.md

```diff
@@ -309,4 +310,97 @@ main.c function function
 
 ** 0 of 1 failed (1 iterations)
 VERIFICATION SUCCESSFUL
-```
\ No newline at end of file
+```
+
+### The memory snapshot harness
+
+The `function-call` harness is used in situations when we want the analysed
+function to work in arbitrary environment. If we want a more restricted
+environment or if we have the program in which our function will only be called,
```

Comment:
remove only

Reply:
I wanted "only" there somewhere. What about:
..or if we have the only program in which our function will be called..
?